### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by Nenad85**
misc fixes

### CODE REVIEW

File: cypress/integration/dib-travel/company-settings/subscription/helpers/confirmation-displaying-first-page.ts [hunk 1]
```diff
@@ -1,6 +1,3 @@
-export const confirmFirstPagePreview = (): void => {
-  cy.get('dib-company-management dib-subscription dib-subscription-purchase-history p').should(
-    'contain',
-    ' Apr 2, 2021 '
-  );
+export const confirmFirstPagePreview = (date: string): void => {
+  cy.get('dib-company-management dib-subscription dib-subscription-purchase-history p').should('contain', date);
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/subscription/helpers/confirmation-displaying-first-page.ts]: This one-liner is not a huge benefit.

You can use `cy.get('dib-company-management dib-subscription dib-subscription-purchase-history p').should('contain', date);` directly

File: cypress/integration/dib-travel/company-settings/subscription.spec.ts [hunk 2]
```diff
@@ -296,9 +312,10 @@ describe('Company Settings - Subscription', () => {
       .should('contain', paymentMethod.primaryContact.email)
       .should('contain', paymentMethod.companyInformation.address)
       .should('contain', 1)
-      .should('contain', paymentMethod.currency.originalCurrency)
-      .should('contain', paymentMethod.invoiceRecipient.email)
-      .should('contain', paymentMethod.invoiceRecipient.vatNumber);
+      .should('contain', paymentMethod.currency.originalCurrency);
+    // TODO: This should be discussed, because on the staging environment, we don't have section "INVOICE RECIPIENT E-MAIL AND VAT NUMBER"
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/subscription.spec.ts:322]: maybe you can add env if/else here? And staging can be clarified later

File: cypress/integration/dib-travel/company-settings/payment-method/billing-profiles.spec.ts [hunk 3]
```diff
@@ -158,12 +162,14 @@ describe('Company Settings - Payment Method - Billing Profiles', () => {
       .should('contain', paymentMethod.primaryContact.modifiedEmail)
       .should('contain', paymentMethod.modifiedCompanyInformation.address)
       .should('contain', 2)
-      .should('contain', paymentMethod.currency.originalCurrency)
-      .should('contain', paymentMethod.invoiceRecipient.modifiedEmail)
-      .should('contain', paymentMethod.invoiceRecipient.modifiedVatNumber);
+      .should('contain', paymentMethod.currency.originalCurrency);
+    // TODO: This should be discussed, because on the staging environment, we don't have section "INVOICE RECIPIENT E-MAIL AND VAT NUMBER"
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/payment-method/billing-profiles.spec.ts:165]: Same as on other place

File: cypress/integration/dib-travel/company-settings/payment-method/billing-profiles.spec.ts [hunk 4]
```diff
@@ -81,18 +81,20 @@ describe('Company Settings - Payment Method - Billing Profiles', () => {
       .should('contain', paymentMethod.primaryContact.email)
       .should('contain', paymentMethod.companyInformation.address)
       .should('contain', 1)
-      .should('contain', paymentMethod.currency.originalCurrency)
-      .should('contain', paymentMethod.invoiceRecipient.email)
+      .should('contain', paymentMethod.currency.originalCurrency);
+    // TODO: This should be discussed, because on the staging environment, we don't have section "INVOICE RECIPIENT E-MAIL AND VAT NUMBER"
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/payment-method/billing-profiles.spec.ts:89]: Same as other place

File: cypress/integration/dib-travel-iframe/company-settings/payment-method/helpers/submit-empty-credit-card-form-and-confirm.ts [hunk 5]
```diff
@@ -2,12 +2,13 @@ export const submitEmptyCreditCardFormAndConfirm = (): void => {
   cy.get('.cdk-overlay-container dib-add-credit-card-dialog ui-button[type=success]').click();
 
   cy.get('.cdk-overlay-container dib-add-credit-card-dialog .error')
-    .should('contain', 'Postal code is required.')
+    // TODO: This should be discussed, because on the staging environment, we don't have section "COMPANY INFORMATION"
```
Comment (sjova) [inline cypress/integration/dib-travel-iframe/company-settings/payment-method/helpers/submit-empty-credit-card-form-and-confirm.ts]: Same as other places.

**Review by sjova — CHANGES_REQUESTED** [review]
ping me when you implement all fixes

**Review by sjova — APPROVED** [review]
Feel free to merge after @stefannikolic approval

**Review by stefannikolic — APPROVED** [review]
Everything is okay.

**Pull request merged**